common/src/main/java/org/axonframework/common/annotation/AnnotationUtils.java

```diff
@@ -314,6 +316,16 @@ public static boolean isAnnotatedWith(Class<? extends Annotation> target,
         return hasSubjectAnnotation;
     }
 
+    /**
+     * Creates a {@link Predicate} that checks whether the given {@link Member} is annotated with the given {@code annotationType}.
+     *
+     * @param annotationType the annotation type to check
```

Comment:
The JavaDoc could use a little love, and some `@Nonnull` annotations would be in place.